docs/configuring-services.md

```diff
@@ -0,0 +1,169 @@
+
+The Kubernetes controller manager is a daemon that embeds the core control
+loops shipped with Kubernetes. In Kubernetes, a controller is a control loop
+which watches the shared state of the cluster through the apiserver and makes
+changes attempting to move the current state towards the desired state.
+
+The kube-controller-manager daemon is started using the arguments in
+`${SNAP_DATA}/args/kube-controller-manager`. For more detail on these
+arguments, see the upstream
+[kube-controller-manager documentation][kube-controller-manager].
+
+### snap.microk8s.daemon-etcd
+
+Etcd is a key/value datastore used to support the components of Kubernetes.
+
+The etcd daemon is started using the arguments in `${SNAP_DATA}/args/etcd`. For
+more information on the configuration, see the [etcd documentation][etcd]. Note
+that different channels of MicroK8s may use different versions of etcd.
+
+
+**Defaults:**
+
+```ỳaml
+--data-dir=${SNAP_COMMON}/var/run/etcd
+--advertise-client-urls=https://${DEFAULT_INTERFACE_IP_ADDR}:12379
+--listen-client-urls=https://0.0.0.0:12379
+--client-cert-auth
+--trusted-ca-file=${SNAP_DATA}/certs/ca.crt
+--cert-file=${SNAP_DATA}/certs/server.crt
+--key-file=${SNAP_DATA}/certs/server.key
+```
+
+### snap.microk8s.daemon-kubelet
+
+The kubelet is the primary “node agent” that runs on each node. The kubelet
+takes a set of PodSpecs(a YAML or JSON object that describes a pod) that are
+provided and ensures that the containers described in those PodSpecs are
+running and healthy. The kubelet doesn’t manage containers which were not
+created by Kubernetes.
+
+The kubelet daemon is started using the arguments in
+`${SNAP_DATA}/args/kubelet`. These are fully documented in the upstream
+[kubelet documentation][kubelet].
+
+### snap.microk8s.daemon-proxy
+
+The Kubernetes network proxy runs on each node. This reflects services as
+defined in the Kubernetes API on each node and can do simple TCP, UDP, and SCTP
+stream forwarding or round robin TCP, UDP, and SCTP forwarding across a set of
+backends.
+
+The kube-proxy daemon is started using the arguments in
+`${SNAP_DATA}/args/kube-proxy`. For more details see the upstream
+[kube-proxy documentation][kube-proxy].
+
+### snap.microk8s.daemon-scheduler
+
+The Kubernetes scheduler is a workload-specific function which takes into
+account individual and collective resource requirements, quality of service
+requirements, hardware/software/policy constraints, affinity and anti-affinity
+specifications, data locality, inter-workload interference, deadlines, and so
+on. Workload-specific requirements will be exposed through the API as
+necessary.
+
+The kube-scheduler daemon started using the arguments in
+`${SNAP_DATA}/args/kube-scheduler`. These are explained fully in the
+upstream [kube-scheduler documentation][kube-scheduler].
+
+
+To reconfigure a service you will need to edit the corresponding file and then restart the respective daemon. For example:
+
+
```

Comment:
I would also add flannel here.

Reply:
I added a flannel section in the appropriate place, please check it out!